### ISSUE TITLE
ghost setup fails on Alpine

### ISSUE THREAD

**Issue by noahwhite** [TryGhost/Ghost-CLI#322]
## Issue Summary

ghost production setup fails on Alpine with the following error:

**Error occurred running command: '/bin/sh -c lsb_release -a'**

## Steps to Reproduce

1. Install ghost-cli on alpine 6.11. I used a docker container and can supply the Dockerfile if needed.
1. Run ghost install local
1. Run ghost setup

Observe the error above is produced. lsb_release is not present on Alpine. Ghost needs to use an alternate method of obtaining this info on Alpine.

## Technical details:

```
OS: Alpine 6.11
Debug Information:
Node Version: v6.11.0
Ghost-CLI Version: 1.0.0-beta.6
Environment: production
Command: 'ghost setup'
```

**Comment by acburdine** [comment]
Thanks for the issue! I realized this might happen today as I was working on another PR, this should be fixed by #318

**Comment by noahwhite** [comment]
Cool, thanks! 

BTW any thoughts on why a local install doesn't seem to be exposed outside a docker container? I'm thinking a local install only binds to localhost vs. 0.0.0.0 in which case that could explain what I am seeing. Sorry to hijack this issue. Happy to open a new one for this if you think its worth it. 

Cheers,

-Noah

**Comment by acburdine** [comment]
Yeah that's probably why. What if you tried `ghost install local --ip 0.0.0.0`?

**Comment by noahwhite** [comment]
Nice, that did it! I had been simply trying to for it with --url=http://0.0.0.0:8080

**Comment by acburdine** [comment]
Closed by #318
